pkg/codegen/testing/test/testdata/configure-prop-names/python/pulumi_gcp/compute/instancebootdiskinitializeparams/outputs.py

```diff
@@ -30,23 +47,23 @@ def __init__(__self__, *,
         """
         InstanceBootDiskInitializeParams._configure(
             lambda key, value: pulumi.set(__self__, key, value),
-            image=image,
+            image_name=image_name,
         )
     @staticmethod
     def _configure(
              _setter: Callable[[Any, Any], None],
-             image: Optional[str] = None,
+             image_name: Optional[str] = None,
              opts: Optional[pulumi.ResourceOptions]=None,
              **kwargs):
-        if 'image' in kwargs:
-            image = kwargs['image']
+        if 'imageName' in kwargs:
+            image_name = kwargs['imageName']
 
-        if image is not None:
-            _setter("image", image)
+        if image_name is not None:
+            _setter("image_name", image_name)
```

Comment:
Orthogonal to this PR (as it's not related to fixing this particular issue), and I didn't noticed it previously, but I am surprised to see `_configure` for the output types. I think we should consider removing it from the output types and back to the way it was before, as a separate PR. `_configure` should only be necessary for inputs.

Reply:
We could open an issue. My only reservation is that it creates 2 different ways that we're setting fields on Input/Output types.